<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Anushri" failures="1" tests="5" name="ChromeTest" time="68.802" errors="0" timestamp="2024-10-16T11:01:37 IST">
  <testcase classname="test.LoginSignUpFacebook" name="validationOfCreateNewAccountButton" time="1.14"/>
  <testcase classname="test.CreateAccountTestClass" name="validationcreateAccountButton" time="2.059"/>
  <testcase classname="test.CreateAccountTestClass" name="verificationOfMetaPayLink" time="10.017"/>
  <testcase classname="test.FriendPageTestClass" name="validationOfFavourites" time="7.322">
    <failure type="java.lang.AssertionError" message="expected [https://static.xx.fbcdn.net/rsrc.php/v3/yU/r/oIIZ26adGMr.png] but found [https://static.xx.fbcdn.net/rsrc.php/v3/y7/r/4Lea07Woawi.png]">
      <![CDATA[java.lang.AssertionError: expected [https://static.xx.fbcdn.net/rsrc.php/v3/yU/r/oIIZ26adGMr.png] but found [https://static.xx.fbcdn.net/rsrc.php/v3/y7/r/4Lea07Woawi.png]
at test.FriendPageTestClass.validationOfFavourites(FriendPageTestClass.java:117)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- validationOfFavourites -->
  <testcase classname="test.FriendPageTestClass" name="validationOfFriendPageURL" time="1.834"/>
  <testcase classname="test.FriendPageTestClass" name="validationOfSendButton" time="0.0">
    <ignored/>
  </testcase> <!-- validationOfSendButton -->
</testsuite> <!-- ChromeTest -->
